graylog2-web-interface/src/views/components/aggregationwizard/ElementsConfiguration.tsx

```diff
@@ -21,7 +21,7 @@ import styled from 'styled-components';
 
 import AggregationWidgetConfig from 'views/logic/aggregationbuilder/AggregationWidgetConfig';
 
-import ElementConfigurationContainer from './elementConfiguration/ElementConfigurationContainer';
+import ElementConfigurationSection from './elementConfigurationSections/ElementConfigurationSection';
```

Comment:
Renaming the directory is not necessary in my opinion. Also, I am getting repeatedly confused by the `elementConfiguration` vs. `aggregationElements` directories. The first contains the components for the configuration form, the latter the definition for its `AggregationElement` type (I had to look it up right now, otherwise I would not have remembered it).

Maybe we should get rid of them and create `group-by`, `metric`, `sort` & `visualization` directories instead, which contain the required components for their configuration and an `index.tsx` exporting their `AggregationElement` definition? This way we would also have a nicer separation, because we know that we do not import anything but that definition from their directories outside of them.

Reply:
I like the idea to create a directory for each aggregation element. I suggest changing this in a follow-up PR, after we've merged https://github.com/Graylog2/graylog2-server/pull/10404 which is based in this PR. For now I just reverted the `elementsConfiguration` directory name change.